packages/@okta/vuepress-site/docs/guides/auth-js/main/index.md

```diff
@@ -4,7 +4,7 @@ title: Auth JS fundamentals
 
 <ApiLifecycle access="ie" /><br>
 
-> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Auth JS fundamentals](/docs/guides/archive-auth-js/main/). See [Identify your Okta solution](https://help.okta.com/okta_help.htm?type=oie&id=ext-oie-version) to determine your Okta version.
+> **Note:** This document is only for Okta Identity Engine. If you’re using Okta Classic Engine, see [Auth JS fundamentals](/docs/guides/archive-auth-js/main/). See [Identify your Okta solution](https://help.okta.com/oie/en-us/Content/Topics/identity-engine/oie-verify-version.htm) to determine your Okta version.
```

Comment:
We should not be hardcoding Product docs URLs. They want us to use aliases.

Reply:
I discussed this with Brian earlier. I agree that using aliases makes sense, but the trouble is that they 404 for me. I've tested it on Firefox/Chrome/Safari on mac and Edge/Firefox on Windows, and the problem is the same everywhere. I also asked a couple of members of my team to test it out, and they reported the same.

I'd rather use the longer link that I know works for now, until we sort out what's going on, and then I'm happy to change to the alias once it is working.

Reply:
All of our docs are using aliases for H.O.C links. I think there was an issue with the H.O.C site over the weekend, but it's resolved now. Are you still getting 404 on the H.O.C. site? If so, you should let them know.

Reply:
OK @vanngo-okta , links reverted back to alias in latest commit. Let me know if that looks OK. Thanks!